src/parser/core/modules/DoTs.ts

```diff
@@ -79,74 +64,63 @@ export abstract class DoTs extends Analyser {
 
 	private onApply(event: Events['statusApply']) {
 		const status = this.data.getStatus(event.status)
-		const statusID = status?.id
-		if (statusID == null) { return }
+		// Cannot track for statuses that are not defined with a duration
+		if (status == null || status.duration == null) { return }
```

Comment:
More of a nit but I wonder if `status?.duration  == null` is cleaner since it accounts for both and is clear that the duration is what we care about. Up to you.